assets/sass/application.scss

```diff
@@ -25,3 +25,7 @@ $govuk-page-width: $moj-page-width;
 @import './global';
 @import './incentive-summary';
 @import './pgd-region';
+
+.govuk-notification-banner p {
```

Comment:
i don't think it's a great idea to force _all_ notification banners to have a different shape from the GDS spec. would be better to add a class specific to _this_ banner. something like `.reviews-policy-notification-banner.govuk-notification-banner p`

Reply:
normally, i'm all for consistency! but i'm hoping that if we use this banner component again elsewhere, we'd use it without modification with respect to what GDS designed